pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -1571,6 +1571,32 @@ func parseU32ConfigItem(newGlobalConfigPtr *types.GlobalConfig, key string,
 	return ""
 }
 
+func parseBoolConfigItem(newGlobalConfigPtr *types.GlobalConfig, key string,
+	value string, varPtr *bool) string {
+	newBool, err := strconv.ParseBool(value)
+	if err != nil {
+		errStr := fmt.Sprintf("bad bool value %s for %s. Error: %s\n",
+			value, key, err)
+		log.Errorf("parseU32ConfigItem: %s", errStr)
+		return errStr
```

Comment:
The way you should return errors in golang is to use the error type, not with a string type.
I think I said that in a previous review already.

Reply:
Are we standardizing it on a per-function level? I somehow missed that..
Will make the change..

Reply:
It's isn't us; it is the standard golang pattern for errors.